backend/api/main.go

```diff
@@ -4,9 +4,10 @@ package main
 import (
 	"log"
 
-	"github.com/abodsakah/BTH-appen/backend/api/src/DB"
-	"github.com/abodsakah/BTH-appen/backend/api/src/Notifications"
-	"github.com/abodsakah/BTH-appen/backend/api/src/Routes"
+	db "github.com/abodsakah/BTH-appen/backend/api/src/DB"
+	notifications "github.com/abodsakah/BTH-appen/backend/api/src/Notifications"
+	routes "github.com/abodsakah/BTH-appen/backend/api/src/Routes"
+	scraper "github.com/abodsakah/BTH-appen/backend/api/src/Scraper"
```

Comment:
```suggestion
	"github.com/abodsakah/BTH-appen/backend/api/src/Scraper"
```

Reply:
I couldn't delete them, I'm getting them back as soon as I save the file!